pkg/common/token.go

```diff
@@ -54,7 +55,7 @@ func GetToken(ResourceId string, i Identity) (r TokenResponse, err error) {
 	// Unmarshall response body into struct
 	err = json.Unmarshal(httpResponseBodyBytes, &r)
 	if err != nil {
-		return r, errors.Wrapf(err, "unmarshalling authentication token response failed")
+		return r, errors.Wrapf(err, "unmarshalling authentication token response failed for %s", httpResponseBodyBytes)

```

Comment:
This http response could include the returned token, which would be sensitive data.